src/main/java/com/sergiofah/integracao/service/Service.java

```diff
@@ -0,0 +1,69 @@
+package com.sergiofah.integracao.service;
+
+import com.sergiofah.integracao.model.Category;
+import com.sergiofah.integracao.model.Line;
+import com.sergiofah.integracao.model.Product;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.net.URL;
+import java.util.List;
+import java.util.Optional;
+
+public class Service {
```

Comment:
In the same thought used in backend, it's necessary to Split this classe into 3 services classes: CategoryService, LineService and ProductService.